app/mailers/user_mailer.rb

```diff
@@ -1,30 +1,13 @@
 class UserMailer < ApplicationMailer
-  def removal_email(user, organisation)
-    translation_root = "mailers.user.#{user.service}.removal_email"
-    mailer_options = {
-      to: user.email,
-      subject: t("#{translation_root}.subject", organisation_name: organisation.name),
-      body: t("#{translation_root}.body",
-              user_name: user.full_name,
-              organisation_name: organisation.name),
-    }
-
-    notify_email(mailer_options)
+  def user_invitation_notification(user, organisation)
```

Comment:
Surely `user` is self explanatory here.

Reply:
This change is based on [this message](https://ukgovernmentdfe.slack.com/archives/C06962474AC/p1705488689303999) - [Direct Link to article](https://katanacode.com/blog/ruby-on-rails-action-mailer-how-to-name-mailer-methods/).

It _could_ be made implicit that the resource being affected is the user themself, but to keep things scalably explicit, prefixing with `user_` identifies that a user is the resource being effected.

For example, notifications about successfully submitted claims would be `UserMailer#claim_submitted_notification`.

Reply:
Hmm, that's true. I'll change and add a conventions doc.

Reply:
Resolved in https://github.com/DFE-Digital/itt-mentor-services/pull/303/commits/5aa71ade825ef6452df26df71a5522b86cdbd56d and https://github.com/DFE-Digital/itt-mentor-services/pull/303/commits/6080bdff0f1e3735847653cdc896f38a8f6b7178 😄